tests/test_project.py

```diff
@@ -171,9 +172,158 @@ def test_generate_no_handlers(project):
     mock_plugin = MagicMock(spec=["generate"])
     with patch.object(project, "_plugin", mock_plugin):
         project.generate()
+        project.generate_docs()
     mock_plugin.generate.assert_called_once_with(project)
 
 
+def test_generate_with_docs(project, tmpdir_factory):
```

Comment:
if you use `tmp_path_factory`, it returns a `Path`

Reply:
Changed per your suggestion.